core/constants.py

```diff
@@ -48,3 +48,7 @@
 YEAR_CHOICES = []
 for year in range(2013, (datetime.now().year + 1)):
     YEAR_CHOICES.append((year, year))
+
+CSV_REQUIRED_HEADING = {'Make', 'Type', 'Asset Code', 'Category', 'Sub-Category'}
+CSV_REQUIRED_HEADING_MODEL = CSV_REQUIRED_HEADING.union({'Model Number'})
+CSV_REQUIRED_HEADING_SERIAL = CSV_REQUIRED_HEADING.union({'Serial No'})
```

Comment:
I'd suggest we also include constants for all fields we have for consistency.
eg
```
MAKE = 'Make'
TYPE = 'Type'... etc

OPTIONAL_IMPORT_FIELDS = ((MAKE, 'Make'), (TYPE, 'Type'), etc)
REQUIRED_IMPORT_FIELDS = ...
```
The required fields should be `Model Number`, and  `Asset Code` or `Serial Number`.
If you wish to follow what you have you can do this but have constants for the field names
```
CSV_OPTIONAL_HEADING = {'Make', 'Type', 'Asset Code', 'Category', 'Sub-Category'}
CSV_REQUIRED_HEADING = {'Model Number'}
CSV_REQUIRED_HEADING_ASSET_CODE = CSV_REQUIRED_HEADING.union({'Asset Code'})
CSV_REQUIRED_HEADING_SERIAL = CSV_REQUIRED_HEADING.union({'Serial Number'})
```
let's also change `HEADING` to `HEADERS`